src/html/partials/story/intro.hbs

```diff
@@ -19,28 +25,28 @@ Popular music is often looked on with cynicism as insipid and endlessly recycled
 
 <div class="prose scrolltext" data-clearsong="1">
 <p class="step unpad-bottom-3">
-The authors analyzed the songs in their dataset using <i>sonic features</i> - that is, properties calculated from the raw audio of the songs.
+	The authors analyzed the sonic features&#8202;—&#8202;—properties like beat, “accousticness,” and “valence,” or mood&#8202;—&8202;for every song that has appeared on the charts since its debut in 1955. The features are calculated directly from the raw audio of the songs.
 </p>
 
 <p class="step unpad-bottom-2" data-song="Believe" data-hide_avg="1">
-Let's go over some of those features. As an example, I'll be using <i>Believe</i> by Cher, which held the #1 spot on the Billboard Hot 100 for four weeks in 1999.
+	Let's go over some of those features. As an example, I'll be using “Believe” by Cher, which held the #1 spot on the Billboard Hot 100 for four weeks in 1999.
```

Comment:
So you'd prefer quotation marks for titles rather than italics? I noticed there are still some titles in italics, but I'm happy to go through and change those.

I went with italics because I was concerned quotation marks looked a little "busy", especially in paragraphs with lots of them, but I don't have a strong opinion on this.

Reply:
Yeah, we use the Chicago Manual of Style at the school and some folks here are kind of sticklers about it, so I updated to match CMOS's preferred styling. I'll definitely take another pass later, so definitely feel free to correct any that you encounter, but I can always fix later as well

Reply:
Ah, I just realized why you probably missed some of the titles.

The copy for the intro section where each of the sonic attributes is explained isn't actually in `intro.hbs`. It's stored in js-land (for no particularly good reason), in the file `src/js/sonic-reveal.js`.

If you were just making your edits based on the html file, you may want to look over that js file.

(I already went ahead and changed the title formatting there, but I was thinking you might have some content edits for that section.)